jenkins/spark-premerge-build.sh

```diff
@@ -160,6 +163,36 @@ rapids_shuffle_smoke_test() {
     $SPARK_HOME/sbin/stop-master.sh
 }
 
+# parameters for Hybrid featrue
+spark_prefix="${SPARK_VER:0:3}" # get prefix from SPARK_VER, e.g.: 3.2, 3.3 ... 3.5
+GLUTEN_BUNDLE_JAR="gluten-velox-bundle-spark${spark_prefix}_2.12-ubuntu_20.04_x86_64-1.2.0.jar"
+HYBRID_JAR="rapids-4-spark-hybrid_2.12-${PROJECT_TEST_VER}.jar"
+GLUTEN_THIRD_PARTY_JAR="gluten-thirdparty-lib-1.2.0-ubuntu-20.04-x86_64.jar"
+
+prepare_for_hybrid_feature() {
+    mvn -B dependency:get -DgroupId=com.nvidia \
+                       -DartifactId=gluten-velox-bundle \
```

Comment:
What artifacts are we going to release?

From the scripts, we'll release `gluten-velox-bundle`, `rapids-4-spark-hybrid_2.12`, `gluten-thirdparty-lib`?

Or we can consider package all this dependencies into one jar.

Reply:
Yes, release 3 jars, we can not package all this dependencies into one jar. Because customer may have there own customized gluten-velox-bundle jar.  E.g.: customer has developed a file system like AFS.

Reply:
We're good to expose this artifacts info, though they're not externally available, 

but @res-life  has documented how to build these artifacts, refer to https://github.com/NVIDIA/spark-rapids/blob/a1bf0a07399e8a208594457d2d492ffd2cffbdd5/integration_tests/README.md#running-with-hybrid-execution